payment/src/main/java/com/kyriba/conference/payment/api/PaymentController.java

```diff
@@ -1,96 +1,60 @@
 package com.kyriba.conference.payment.api;
 
-import com.fasterxml.jackson.annotation.JsonFormat;
 import com.kyriba.conference.payment.api.dto.PaymentDto;
-import com.kyriba.conference.payment.api.dto.PaymentUpdateParamsDto;
+import com.kyriba.conference.payment.api.dto.PaymentStatusUpdateDto;
+import com.kyriba.conference.payment.api.dto.Receipt;
 import com.kyriba.conference.payment.api.dto.TicketDto;
-import com.kyriba.conference.payment.domain.PaymentStatus;
-import com.kyriba.conference.payment.domain.dto.Amount;
-import io.swagger.annotations.ApiModel;
+import com.kyriba.conference.payment.service.PaymentService;
 import io.swagger.annotations.ApiParam;
-import lombok.Value;
+import lombok.AllArgsConstructor;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
-import javax.validation.constraints.NotNull;
-import java.math.BigDecimal;
-import java.time.LocalDateTime;
-import java.time.Month;
-import java.util.Collections;
-import java.util.Currency;
 import java.util.List;
-import java.util.Locale;
 
-import static com.kyriba.conference.payment.domain.PaymentMethodType.CREDIT_CARD;
-import static com.kyriba.conference.payment.domain.PaymentStatus.PENDING;
 import static org.springframework.http.MediaType.APPLICATION_JSON_UTF8_VALUE;
 
 
 /**
  * @author Igor Lizura
  */
+@AllArgsConstructor
 @RestController
 @RequestMapping("/api/v1/payments")
 public class PaymentController {
+    private final PaymentService paymentService;
+
     @ResponseStatus(HttpStatus.OK)
     @GetMapping
     @Valid List<PaymentDto> getPayments() {
```

Comment:
@xpinjection should we validate response?

Reply:
This validation will not work at all.

Reply:
fixed